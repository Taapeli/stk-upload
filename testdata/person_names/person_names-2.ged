0 HEAD
1 CHAR UTF-8
0 @I0 INDI
1 NAME Matti/Mattila/
0 @I1 INDI
1 NAME Matti/Frisk os. Mattila/
0 @I2 INDI
1 NAME Matti/Reipas e. Frisk os. Laine/
0 @I3 INDI
1 NAME Matti/Lehti os. Lampi e. af Damm/
0 @I4 INDI
1 NAME Matti/Mattila (Matts)/
0 @I5 INDI
1 NAME Matti/Hällström (af Hällström)/
0 @I6 INDI
1 NAME Matti/Hällström (af Hällström) e. Hellström, os. Mönkkönen/
0 @I7 INDI
1 NAME Matti/af Hällström (Hällström) e. Hellström, os. Mönkkönen/
0 @I8 INDI
1 NAME Matti/Garcia Marquez/
0 @I9 INDI
1 NAME Matti/Heikkilä/Mattila/
0 @I10 INDI
1 NAME Matti/Heikkilä,Mattila/
0 @I11 INDI
1 NAME Matti/von der Goltz/
0 @I12 INDI
1 NAME Matti/de Godzinsky/
0 @I13 INDI
1 NAME Matti Pekka/Mattila/
0 @I26 INDI
1 NAME Matti*/Mattila/
0 @I39 INDI
1 NAME Matti Pekka*/Mattila/
0 @I52 INDI
1 NAME Matti* Pekka/Mattila/
0 @I65 INDI
1 NAME Matti (Pekka)/Mattila/
0 @I78 INDI
1 NAME (Matti) Pekka/Mattila/
0 @I91 INDI
1 NAME (Matti)/Mattila/
0 @I104 INDI
1 NAME Matti Pekanpoika/Mattila/
0 @I105 INDI
1 NAME Matti Pettersson/Mattila/
0 @I106 INDI
1 NAME Matti Pettersson/Mattila/Andersson
0 @I117 INDI
1 NAME poikalapsi/Mattila/
0 @I130 INDI
1 NAME Johan Henrici/Mattila/
0 @I143 INDI
1 NAME Johan//Mattsson
0 @I143 INDI
1 NAME poika//Mattsson
0 @I144 INDI
1 NAME poikavauva//Mattsson
0 @I145 INDI
1 NAME Pauli/Frimodig/
0 @P1@ INDI
1 NAME RITVA PIRJO H./JYLHÄ\SINISALO\TOIVONEN/ 
0 TRLR
