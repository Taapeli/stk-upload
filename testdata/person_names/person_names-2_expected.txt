0 HEAD
1 CHAR UTF-8
0 @PERSON1@ INDI
1 NAME RITVA PIRJO H./JYLHÄ\SINISALO\TOIVONEN/ 
1 SEX F
0 @I0 INDI
1 NAME Matti/Mattila/
0 @I1 INDI
1 NAME Matti/Mattila/
2 TYPE birth
1 NAME Matti/Frisk/
0 @I2 INDI
1 NAME Matti/Laine/
2 TYPE birth
1 NAME Matti/Reipas/
1 NAME Matti/Frisk/
2 TYPE entinen
0 @I3 INDI
1 NAME Matti/Lampi/
2 TYPE birth
1 NAME Matti/Lehti/
1 NAME Matti/af Damm/
2 TYPE entinen
0 @I4 INDI
1 NAME Matti/Mattila/
1 NAME Matti/Matts/
2 TYPE aka
0 @I5 INDI
1 NAME Matti/Hällström/
1 NAME Matti/af Hällström/
2 TYPE aka
0 @I6 INDI
1 NAME Matti/Mönkkönen/
2 TYPE birth
1 NAME Matti/Hällström/
1 NAME Matti/af Hällström/
2 TYPE aka
1 NAME Matti/Hellström/
2 TYPE entinen
0 @I7 INDI
1 NAME Matti/Mönkkönen/
2 TYPE birth
1 NAME Matti/af Hällström/
1 NAME Matti/Hällström/
2 TYPE aka
1 NAME Matti/Hellström/
2 TYPE entinen
0 @I8 INDI
1 NAME Matti/Garcia Marquez/
0 @I9 INDI
1 NAME Matti/Heikkilä/
1 NAME Matti/Mattila/
2 TYPE aka
0 @I10 INDI
1 NAME Matti/Heikkilä/
1 NAME Matti/Mattila/
2 TYPE aka
0 @I11 INDI
1 NAME Matti/von der Goltz/
0 @I12 INDI
1 NAME Matti/de Godzinsky/
0 @I13 INDI
1 NAME Matti Pekka/Mattila/
0 @I14 INDI
1 NAME Matti Pekka/Mattila/
2 TYPE birth
1 NAME Matti Pekka/Frisk/
0 @I15 INDI
1 NAME Matti Pekka/Laine/
2 TYPE birth
1 NAME Matti Pekka/Reipas/
1 NAME Matti Pekka/Frisk/
2 TYPE entinen
0 @I16 INDI
1 NAME Matti Pekka/Lampi/
2 TYPE birth
1 NAME Matti Pekka/Lehti/
1 NAME Matti Pekka/af Damm/
2 TYPE entinen
0 @I17 INDI
1 NAME Matti Pekka/Mattila/
1 NAME Matti Pekka/Matts/
2 TYPE aka
0 @I18 INDI
1 NAME Matti Pekka/Hällström/
1 NAME Matti Pekka/af Hällström/
2 TYPE aka
0 @I19 INDI
1 NAME Matti Pekka/Mönkkönen/
2 TYPE birth
1 NAME Matti Pekka/Hällström/
1 NAME Matti Pekka/af Hällström/
2 TYPE aka
1 NAME Matti Pekka/Hellström/
2 TYPE entinen
0 @I20 INDI
1 NAME Matti Pekka/Mönkkönen/
2 TYPE birth
1 NAME Matti Pekka/af Hällström/
1 NAME Matti Pekka/Hällström/
2 TYPE aka
1 NAME Matti Pekka/Hellström/
2 TYPE entinen
0 @I21 INDI
1 NAME Matti Pekka/Garcia Marquez/
0 @I22 INDI
1 NAME Matti Pekka/Heikkilä/
1 NAME Matti Pekka/Mattila/
2 TYPE aka
0 @I23 INDI
1 NAME Matti Pekka/Heikkilä/
1 NAME Matti Pekka/Mattila/
2 TYPE aka
0 @I24 INDI
1 NAME Matti Pekka/von der Goltz/
0 @I25 INDI
1 NAME Matti Pekka/de Godzinsky/
0 @I26 INDI
1 NAME Matti/Mattila/
2 CALL Matti
0 @I27 INDI
1 NAME Matti/Mattila/
2 TYPE birth
2 CALL Matti
1 NAME Matti/Frisk/
0 @I28 INDI
1 NAME Matti/Laine/
2 TYPE birth
2 CALL Matti
1 NAME Matti/Reipas/
1 NAME Matti/Frisk/
2 TYPE entinen
0 @I29 INDI
1 NAME Matti/Lampi/
2 TYPE birth
2 CALL Matti
1 NAME Matti/Lehti/
1 NAME Matti/af Damm/
2 TYPE entinen
0 @I30 INDI
1 NAME Matti/Mattila/
2 CALL Matti
1 NAME Matti/Matts/
2 TYPE aka
0 @I31 INDI
1 NAME Matti/Hällström/
2 CALL Matti
1 NAME Matti/af Hällström/
2 TYPE aka
0 @I32 INDI
1 NAME Matti/Mönkkönen/
2 TYPE birth
2 CALL Matti
1 NAME Matti/Hällström/
1 NAME Matti/af Hällström/
2 TYPE aka
1 NAME Matti/Hellström/
2 TYPE entinen
0 @I33 INDI
1 NAME Matti/Mönkkönen/
2 TYPE birth
2 CALL Matti
1 NAME Matti/af Hällström/
1 NAME Matti/Hällström/
2 TYPE aka
1 NAME Matti/Hellström/
2 TYPE entinen
0 @I34 INDI
1 NAME Matti/Garcia Marquez/
2 CALL Matti
0 @I35 INDI
1 NAME Matti/Heikkilä/
2 CALL Matti
1 NAME Matti/Mattila/
2 TYPE aka
0 @I36 INDI
1 NAME Matti/Heikkilä/
2 CALL Matti
1 NAME Matti/Mattila/
2 TYPE aka
0 @I37 INDI
1 NAME Matti/von der Goltz/
2 CALL Matti
0 @I38 INDI
1 NAME Matti/de Godzinsky/
2 CALL Matti
0 @I39 INDI
1 NAME Matti Pekka/Mattila/
2 CALL Pekka
0 @I40 INDI
1 NAME Matti Pekka/Mattila/
2 TYPE birth
2 CALL Pekka
1 NAME Matti Pekka/Frisk/
0 @I41 INDI
1 NAME Matti Pekka/Laine/
2 TYPE birth
2 CALL Pekka
1 NAME Matti Pekka/Reipas/
1 NAME Matti Pekka/Frisk/
2 TYPE entinen
0 @I42 INDI
1 NAME Matti Pekka/Lampi/
2 TYPE birth
2 CALL Pekka
1 NAME Matti Pekka/Lehti/
1 NAME Matti Pekka/af Damm/
2 TYPE entinen
0 @I43 INDI
1 NAME Matti Pekka/Mattila/
2 CALL Pekka
1 NAME Matti Pekka/Matts/
2 TYPE aka
0 @I44 INDI
1 NAME Matti Pekka/Hällström/
2 CALL Pekka
1 NAME Matti Pekka/af Hällström/
2 TYPE aka
0 @I45 INDI
1 NAME Matti Pekka/Mönkkönen/
2 TYPE birth
2 CALL Pekka
1 NAME Matti Pekka/Hällström/
1 NAME Matti Pekka/af Hällström/
2 TYPE aka
1 NAME Matti Pekka/Hellström/
2 TYPE entinen
0 @I46 INDI
1 NAME Matti Pekka/Mönkkönen/
2 TYPE birth
2 CALL Pekka
1 NAME Matti Pekka/af Hällström/
1 NAME Matti Pekka/Hällström/
2 TYPE aka
1 NAME Matti Pekka/Hellström/
2 TYPE entinen
0 @I47 INDI
1 NAME Matti Pekka/Garcia Marquez/
2 CALL Pekka
0 @I48 INDI
1 NAME Matti Pekka/Heikkilä/
2 CALL Pekka
1 NAME Matti Pekka/Mattila/
2 TYPE aka
0 @I49 INDI
1 NAME Matti Pekka/Heikkilä/
2 CALL Pekka
1 NAME Matti Pekka/Mattila/
2 TYPE aka
0 @I50 INDI
1 NAME Matti Pekka/von der Goltz/
2 CALL Pekka
0 @I51 INDI
1 NAME Matti Pekka/de Godzinsky/
2 CALL Pekka
0 @I52 INDI
1 NAME Matti Pekka/Mattila/
2 CALL Matti
0 @I53 INDI
1 NAME Matti Pekka/Mattila/
2 TYPE birth
2 CALL Matti
1 NAME Matti Pekka/Frisk/
0 @I54 INDI
1 NAME Matti Pekka/Laine/
2 TYPE birth
2 CALL Matti
1 NAME Matti Pekka/Reipas/
1 NAME Matti Pekka/Frisk/
2 TYPE entinen
0 @I55 INDI
1 NAME Matti Pekka/Lampi/
2 TYPE birth
2 CALL Matti
1 NAME Matti Pekka/Lehti/
1 NAME Matti Pekka/af Damm/
2 TYPE entinen
0 @I56 INDI
1 NAME Matti Pekka/Mattila/
2 CALL Matti
1 NAME Matti Pekka/Matts/
2 TYPE aka
0 @I57 INDI
1 NAME Matti Pekka/Hällström/
2 CALL Matti
1 NAME Matti Pekka/af Hällström/
2 TYPE aka
0 @I58 INDI
1 NAME Matti Pekka/Mönkkönen/
2 TYPE birth
2 CALL Matti
1 NAME Matti Pekka/Hällström/
1 NAME Matti Pekka/af Hällström/
2 TYPE aka
1 NAME Matti Pekka/Hellström/
2 TYPE entinen
0 @I59 INDI
1 NAME Matti Pekka/Mönkkönen/
2 TYPE birth
2 CALL Matti
1 NAME Matti Pekka/af Hällström/
1 NAME Matti Pekka/Hällström/
2 TYPE aka
1 NAME Matti Pekka/Hellström/
2 TYPE entinen
0 @I60 INDI
1 NAME Matti Pekka/Garcia Marquez/
2 CALL Matti
0 @I61 INDI
1 NAME Matti Pekka/Heikkilä/
2 CALL Matti
1 NAME Matti Pekka/Mattila/
2 TYPE aka
0 @I62 INDI
1 NAME Matti Pekka/Heikkilä/
2 CALL Matti
1 NAME Matti Pekka/Mattila/
2 TYPE aka
0 @I63 INDI
1 NAME Matti Pekka/von der Goltz/
2 CALL Matti
0 @I64 INDI
1 NAME Matti Pekka/de Godzinsky/
2 CALL Matti
0 @I65 INDI
1 NAME Matti/Mattila/
2 NICK Pekka
0 @I66 INDI
1 NAME Matti/Mattila/
2 TYPE birth
2 NICK Pekka
1 NAME Matti/Frisk/
0 @I67 INDI
1 NAME Matti/Laine/
2 TYPE birth
2 NICK Pekka
1 NAME Matti/Reipas/
1 NAME Matti/Frisk/
2 TYPE entinen
0 @I68 INDI
1 NAME Matti/Lampi/
2 TYPE birth
2 NICK Pekka
1 NAME Matti/Lehti/
1 NAME Matti/af Damm/
2 TYPE entinen
0 @I69 INDI
1 NAME Matti/Mattila/
2 NICK Pekka
1 NAME Matti/Matts/
2 TYPE aka
0 @I70 INDI
1 NAME Matti/Hällström/
2 NICK Pekka
1 NAME Matti/af Hällström/
2 TYPE aka
0 @I71 INDI
1 NAME Matti/Mönkkönen/
2 TYPE birth
2 NICK Pekka
1 NAME Matti/Hällström/
1 NAME Matti/af Hällström/
2 TYPE aka
1 NAME Matti/Hellström/
2 TYPE entinen
0 @I72 INDI
1 NAME Matti/Mönkkönen/
2 TYPE birth
2 NICK Pekka
1 NAME Matti/af Hällström/
1 NAME Matti/Hällström/
2 TYPE aka
1 NAME Matti/Hellström/
2 TYPE entinen
0 @I73 INDI
1 NAME Matti/Garcia Marquez/
2 NICK Pekka
0 @I74 INDI
1 NAME Matti/Heikkilä/
2 NICK Pekka
1 NAME Matti/Mattila/
2 TYPE aka
0 @I75 INDI
1 NAME Matti/Heikkilä/
2 NICK Pekka
1 NAME Matti/Mattila/
2 TYPE aka
0 @I76 INDI
1 NAME Matti/von der Goltz/
2 NICK Pekka
0 @I77 INDI
1 NAME Matti/de Godzinsky/
2 NICK Pekka
0 @I78 INDI
1 NAME Pekka/Mattila/
2 NICK Matti
0 @I79 INDI
1 NAME Pekka/Mattila/
2 TYPE birth
2 NICK Matti
1 NAME Pekka/Frisk/
0 @I80 INDI
1 NAME Pekka/Laine/
2 TYPE birth
2 NICK Matti
1 NAME Pekka/Reipas/
1 NAME Pekka/Frisk/
2 TYPE entinen
0 @I81 INDI
1 NAME Pekka/Lampi/
2 TYPE birth
2 NICK Matti
1 NAME Pekka/Lehti/
1 NAME Pekka/af Damm/
2 TYPE entinen
0 @I82 INDI
1 NAME Pekka/Mattila/
2 NICK Matti
1 NAME Pekka/Matts/
2 TYPE aka
0 @I83 INDI
1 NAME Pekka/Hällström/
2 NICK Matti
1 NAME Pekka/af Hällström/
2 TYPE aka
0 @I84 INDI
1 NAME Pekka/Mönkkönen/
2 TYPE birth
2 NICK Matti
1 NAME Pekka/Hällström/
1 NAME Pekka/af Hällström/
2 TYPE aka
1 NAME Pekka/Hellström/
2 TYPE entinen
0 @I85 INDI
1 NAME Pekka/Mönkkönen/
2 TYPE birth
2 NICK Matti
1 NAME Pekka/af Hällström/
1 NAME Pekka/Hällström/
2 TYPE aka
1 NAME Pekka/Hellström/
2 TYPE entinen
0 @I86 INDI
1 NAME Pekka/Garcia Marquez/
2 NICK Matti
0 @I87 INDI
1 NAME Pekka/Heikkilä/
2 NICK Matti
1 NAME Pekka/Mattila/
2 TYPE aka
0 @I88 INDI
1 NAME Pekka/Heikkilä/
2 NICK Matti
1 NAME Pekka/Mattila/
2 TYPE aka
0 @I89 INDI
1 NAME Pekka/von der Goltz/
2 NICK Matti
0 @I90 INDI
1 NAME Pekka/de Godzinsky/
2 NICK Matti
0 @I91 INDI
1 NAME /Mattila/
2 NICK Matti
0 @I92 INDI
1 NAME /Mattila/
2 TYPE birth
2 NICK Matti
1 NAME /Frisk/
0 @I93 INDI
1 NAME /Laine/
2 TYPE birth
2 NICK Matti
1 NAME /Reipas/
1 NAME /Frisk/
2 TYPE entinen
0 @I94 INDI
1 NAME /Lampi/
2 TYPE birth
2 NICK Matti
1 NAME /Lehti/
1 NAME /af Damm/
2 TYPE entinen
0 @I95 INDI
1 NAME /Mattila/
2 NICK Matti
1 NAME /Matts/
2 TYPE aka
0 @I96 INDI
1 NAME /Hällström/
2 NICK Matti
1 NAME /af Hällström/
2 TYPE aka
0 @I97 INDI
1 NAME /Mönkkönen/
2 TYPE birth
2 NICK Matti
1 NAME /Hällström/
1 NAME /af Hällström/
2 TYPE aka
1 NAME /Hellström/
2 TYPE entinen
0 @I98 INDI
1 NAME /Mönkkönen/
2 TYPE birth
2 NICK Matti
1 NAME /af Hällström/
1 NAME /Hällström/
2 TYPE aka
1 NAME /Hellström/
2 TYPE entinen
0 @I98a INDI
1 NAME /Hällström/
2 TYPE birth
2 NICK Matti
1 NAME /af Hällström/
1 NAME /Hellström/
2 TYPE entinen
0 @I99 INDI
1 NAME /Garcia Marquez/
2 NICK Matti
0 @I100 INDI
1 NAME /Heikkilä/
2 NICK Matti
1 NAME /Mattila/
2 TYPE aka
0 @I101 INDI
1 NAME /Heikkilä/
2 NICK Matti
1 NAME /Mattila/
2 TYPE aka
0 @I102 INDI
1 NAME /von der Goltz/
2 NICK Matti
0 @I103 INDI
1 NAME /de Godzinsky/
2 NICK Matti
0 @I104 INDI
1 NAME Matti/Mattila/Pekanpoika
0 @I105 INDI
1 NAME Matti/Mattila/Pekanpoika
2 TYPE birth
1 NAME Matti/Frisk/Pekanpoika
0 @I106 INDI
1 NAME Matti/Laine/Pekanpoika
2 TYPE birth
1 NAME Matti/Reipas/Pekanpoika
1 NAME Matti/Frisk/Pekanpoika
2 TYPE entinen
0 @I107 INDI
1 NAME Matti/Lampi/Pekanpoika
2 TYPE birth
1 NAME Matti/Lehti/Pekanpoika
1 NAME Matti/af Damm/Pekanpoika
2 TYPE entinen
0 @I108 INDI
1 NAME Matti/Mattila/Pekanpoika
1 NAME Matti/Matts/Pekanpoika
2 TYPE aka
0 @I109 INDI
1 NAME Matti/Hällström/Pekanpoika
1 NAME Matti/af Hällström/Pekanpoika
2 TYPE aka
0 @I110 INDI
1 NAME Matti/Mönkkönen/Pekanpoika
2 TYPE birth
1 NAME Matti/Hällström/Pekanpoika
1 NAME Matti/af Hällström/Pekanpoika
2 TYPE aka
1 NAME Matti/Hellström/Pekanpoika
2 TYPE entinen
0 @I111 INDI
1 NAME Matti/Mönkkönen/Pekanpoika
2 TYPE birth
1 NAME Matti/af Hällström/Pekanpoika
1 NAME Matti/Hällström/Pekanpoika
2 TYPE aka
1 NAME Matti/Hellström/Pekanpoika
2 TYPE entinen
0 @I112 INDI
1 NAME Matti/Garcia Marquez/Pekanpoika
0 @I113 INDI
1 NAME Matti/Heikkilä/Pekanpoika
1 NAME Matti/Mattila/Pekanpoika
2 TYPE aka
0 @I114 INDI
1 NAME Matti/Heikkilä/Pekanpoika
1 NAME Matti/Mattila/Pekanpoika
2 TYPE aka
0 @I115 INDI
1 NAME Matti/von der Goltz/Pekanpoika
0 @I116 INDI
1 NAME Matti/de Godzinsky/Pekanpoika
0 @I117 INDI
1 NAME N/Mattila/
0 @I118 INDI
1 NAME N/Mattila/
2 TYPE birth
1 NAME N/Frisk/
0 @I119 INDI
1 NAME N/Laine/
2 TYPE birth
1 NAME N/Reipas/
1 NAME N/Frisk/
2 TYPE entinen
0 @I120 INDI
1 NAME N/Lampi/
2 TYPE birth
1 NAME N/Lehti/
1 NAME N/af Damm/
2 TYPE entinen
0 @I121 INDI
1 NAME N/Mattila/
1 NAME N/Matts/
2 TYPE aka
0 @I122 INDI
1 NAME N/Hällström/
1 NAME N/af Hällström/
2 TYPE aka
0 @I123 INDI
1 NAME N/Mönkkönen/
2 TYPE birth
1 NAME N/Hällström/
1 NAME N/af Hällström/
2 TYPE aka
1 NAME N/Hellström/
2 TYPE entinen
0 @I124 INDI
1 NAME N/Mönkkönen/
2 TYPE birth
1 NAME N/af Hällström/
1 NAME N/Hällström/
2 TYPE aka
1 NAME N/Hellström/
2 TYPE entinen
0 @I125 INDI
1 NAME N/Garcia Marquez/
0 @I126 INDI
1 NAME N/Heikkilä/
1 NAME N/Mattila/
2 TYPE aka
0 @I127 INDI
1 NAME N/Heikkilä/
1 NAME N/Mattila/
2 TYPE aka
0 @I128 INDI
1 NAME N/von der Goltz/
0 @I129 INDI
1 NAME N/de Godzinsky/
0 @I130 INDI
1 NAME Johan/Mattila/Henrici
0 @I131 INDI
1 NAME Johan/Mattila/Henrici
2 TYPE birth
1 NAME Johan/Frisk/Henrici
0 @I132 INDI
1 NAME Johan/Laine/Henrici
2 TYPE birth
1 NAME Johan/Reipas/Henrici
1 NAME Johan/Frisk/Henrici
2 TYPE entinen
0 @I133 INDI
1 NAME Johan/Lampi/Henrici
2 TYPE birth
1 NAME Johan/Lehti/Henrici
1 NAME Johan/af Damm/Henrici
2 TYPE entinen
0 @I134 INDI
1 NAME Johan/Mattila/Henrici
1 NAME Johan/Matts/Henrici
2 TYPE aka
0 @I135 INDI
1 NAME Johan/Hällström/Henrici
1 NAME Johan/af Hällström/Henrici
2 TYPE aka
0 @I136 INDI
1 NAME Johan/Mönkkönen/Henrici
2 TYPE birth
1 NAME Johan/Hällström/Henrici
1 NAME Johan/af Hällström/Henrici
2 TYPE aka
1 NAME Johan/Hellström/Henrici
2 TYPE entinen
0 @I137 INDI
1 NAME Johan/Mönkkönen/Henrici
2 TYPE birth
1 NAME Johan/af Hällström/Henrici
1 NAME Johan/Hällström/Henrici
2 TYPE aka
1 NAME Johan/Hellström/Henrici
2 TYPE entinen
0 @I138 INDI
1 NAME Johan/Garcia Marquez/Henrici
0 @I139 INDI
1 NAME Johan/Heikkilä/Henrici
1 NAME Johan/Mattila/Henrici
2 TYPE aka
0 @I140 INDI
1 NAME Johan/Heikkilä/Henrici
1 NAME Johan/Mattila/Henrici
2 TYPE aka
0 @I141 INDI
1 NAME Johan/von der Goltz/Henrici
0 @I142 INDI
1 NAME Johan/de Godzinsky/Henrici
0 @I143 INDI
1 NAME Johan//Mattsson
0 @I143 INDI
1 NAME poika//Mattsson
0 @I144 INDI
1 NAME N//Mattsson
0 @I145 INDI
1 NAME /Frimodig/Pauli
0 TRLR
